Update ja-JP using v2 SDK

Kazushige-Yoshida-A opened the pull request:
Updated ja-JP branch based on the newest en-US branch using v2 SDK

Kazushige-Yoshida-A committed:
ja-JP-v2

Update based on the newest en-US branch

SleepyDeveloper commented on .ask/config:
```diff
@@ -2,18 +2,7 @@
-      "was_cloned": false,
-      "merge": {
-        "skillManifest": {
-          "apis": {
```
@Kazushige-Yoshida-A  @franklin-lobb will removing this from the config file prevent the lambda function from being created?

SleepyDeveloper replied:
I confirmed that this is an issue. 

Attempting to run `ask deploy` without this information in the .ask/config file causes the cli to fail uploading the content.

I made the change locally and will merge.

SleepyDeveloper commented on Readme.md:
```diff
@@ -1,35 +1,25 @@
-
- このチュートリアルを修了すると、以下のことができるようになります。
+**開始** するには次のボタンをクリックしてください。
+[![始めましょう](https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/jp/tutorials/general/buttons/button_get_started.png)](instructions/1-voice-user-interface.md)
```
This image should be on it's own line.

SleepyDeveloper reviewed (commented):
Please fix the README.md

Kazushige-Yoshida-A committed:
Fix bugs in README.md

Fix the issue that the image shouldn't be appeared on it's own line. I also found that its title was incorrect
